### ISSUE TITLE
do not work with latest version of crabgrass

### ISSUE THREAD

**Issue by dauge** [meitar/crabgrass-tools#9]
I use crabgrass-tool for daily backup. Since 31 January, I can't get a successfull backup. MAybe it's related with upgrade to rails 4.2 ?

`cg-make-snapshot.sh version 0.0.2`

```
xxxxx@yyy:~$ /opt/crabgrass-tools/cg-make-snapshot.sh --user _user_ --password _password_ --subgroup --download-directory /home/xxxxx/ _group1_ _group2_

Logging in as _user_...
Finding subgroups for _group1_...
Mirroring group _group1_ with all subgroups:
--2017-02-03 17:20:02--  https://we.riseup.net/_group1_/
Résolution de we.riseup.net (we.riseup.net)… 198.252.153.99
Connexion à we.riseup.net (we.riseup.net)|198.252.153.99|:443… connecté.
requête HTTP transmise, en attente de la réponse… 404 Not Found
2017-02-03 17:20:03 erreur 404 : Not Found.

0 fichiers convertis en 0 secondes.
Finding subgroups for _group2_...
Mirroring group _group2_ with all subgroups:
--2017-02-03 17:20:03--  https://we.riseup.net/_group2_/
Résolution de we.riseup.net (we.riseup.net)… 198.252.153.99
Connexion à we.riseup.net (we.riseup.net)|198.252.153.99|:443… connecté.
requête HTTP transmise, en attente de la réponse… 404 Not Found
2017-02-03 17:20:04 erreur 404 : Not Found.

0 fichiers convertis en 0 secondes.

```

**Comment by irenenirgendwo** [comment]
I have the same problem.

**Comment by grmpflprmpfl** [comment]
please try my fix: https://github.com/grmpflprmpfl/crabgrass-tools/blob/master/cg-make-snapshot.sh

i have also issued a pull request

**Comment by dauge** [comment]
fix work for me, thx

**Comment by fabacab** [comment]
@grmpflprmpfl Thanks for this! I'll confirm this for myself soon (next week?) and merge when I do. Please nudge me politely if I lose track of this, got a lot on my plate right now.

**Comment by fabacab** [comment]
Fixed in 9e49d4377b3a9f8e5e143e8b5c0b1c98af571bba. Props @grmpflprmpfl.
